### PR TITLE
Remove SSH known_hosts logging

### PR DESCRIPTION
**Pull request by beckerhe**
We added that debug logging to track down the issue we had on Stadia with malformed known_hosts file. But this bug has been fixed and is also not relevant anymore, so let's remove this debug logging.

### CODE REVIEW

File: src/OrbitSsh/Session.cpp [hunk 1]
```diff
@@ -105,14 +81,11 @@ outcome::result<void> Session::MatchKnownHosts(const AddrAndPort& addr_and_port,
 
   const int check_result = libssh2_knownhost_checkp(
       known_hosts, addr_and_port.addr.c_str(), addr_and_port.port, fingerprint, fingerprint_length,
-      LIBSSH2_KNOWNHOST_TYPE_PLAIN | LIBSSH2_KNOWNHOST_KEYENC_RAW |
-          ((fingerprint_type + 1) << LIBSSH2_KNOWNHOST_KEY_SHIFT),
```
Comment (beckerhe) [inline src/OrbitSsh/Session.cpp:109]: This is an unrelated change that I just sneaked in. The flags parameter in `libssh2_knownhost_checkp` doesn't accept a fingerprint type. It just happened to not clash with other flags in the bitmask, that's why it worked in the first place.

**Review by florian-kuebler — APPROVED** [review]
